### ISSUE TITLE
MACRO - User input prompt 

### ISSUE THREAD

**Issue by rgodin974** [cncjs/cncjs#616]
Is it possible into the macro widget to have an user input prompt ? It could be very useful to increase the macro’s possibilities. The functionality is in bCNC. Thank you.

**Comment by neilferreri** [comment]
That would be cool!

**Comment by aolshove** [comment]
Yes please!

**Comment by mmdoogie** [comment]
@aolshove PR #615 would let messages like that be done at least, let me know what you think of that implementation for the messages?  There's some screenshots over there.

**Comment by aolshove** [comment]
Yes I think it might.  My only concern is that the M0 throws up a dialog that may block the display of the message?  It seems like a parameter included in the M0 command for a "message" might be a better solution.  Just a though.  I would use the new MSG gcode command when it's implemented though.

**Comment by mmdoogie** [comment]
@aolshove agree, it would be cool to see both.  Should not get covered up though, except on maybe a really small screen.

![msg-with-pause](https://user-images.githubusercontent.com/24367033/86315437-c1218a00-bbef-11ea-9133-c20cef0ac2eb.PNG)
